DigitalLearningSolutions.Data.Tests/DataServices/CourseAdminFieldsDataServiceTests.cs

```diff
@@ -75,6 +80,36 @@ public void GetPromptNameForCourseAndPromptNumber_returns_expected_prompt_name()
             result.Should().BeEquivalentTo("System Access Granted");
         }
 
+        [Test]
+        public void UpdateCustomPromptForCourse_correctly_adds_custom_prompt()
+        {
+            using var transaction = new TransactionScope();
+            try
+            {
+                // Given
+                const string? options = "options";
+
+                // When
+                courseAdminFieldsDataService.UpdateCustomPromptForCourse(100, 3, 1, options);
+                var courseCustomPrompts = courseAdminFieldsDataService.GetCourseAdminFields(100, 101, 2);
+                var customPrompt = courseAdminFieldsDataService.GetCoursePromptsAlphabetical()
+                    .Single(c => c.Item1 == 1)
+                    .Item2;

```

Comment:
Instead of Item1 & Item2, we should name these id and name

Reply:
In C# you can specify tuple field names - https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/value-tuples#tuple-field-names

We're doing this in various places, e.g. see GetUsersByUsername in userService :)

Reply:
renamed to id and name